lib/discourse_api/api/uploads.rb

```diff
@@ -8,6 +8,14 @@ def upload_post_image(args)
                   .to_h
         post('/uploads', args)
       end
+
+      def upload_file(args)
```

Comment:
I'm okay with breaking changes 😉 !! I still think this is beta software since it is 0.x  and I'm trying to follow semver and I think upload_post_image was just poorly named so lets replace it with just upload_file (an image is a file).